server/src/sim.rs

```diff
@@ -204,9 +212,3 @@ fn dump_entity(world: &hecs::World, entity: Entity) -> Vec<Component> {
     }
     components
 }
-
-struct Character {
```

Comment:
Since this is being removed in favor of using `proto::Character`, which we're importing and referencing without qualification, we should take care to replace existing qualified references to `proto::Character` (e.g. in `dump_entity`) with plain `Character` to avoid giving the impression of multiple types. Further, `dump_entity` could now be `Clone`ing the component directly rather than enumerating its fields.